lib/routes/visualRulesController.js

```diff
@@ -127,7 +127,6 @@ function PutVR(req, resp) {
             return myutils.respond(resp, err, data);
         }
         if (data && data.name) {
-            //resp.location(req.url.replace(new RegExp(req.params.id), '')+encodeURIComponent(data.name));
```

Comment:
@AlvaroVega maybe this is an old leftover? It's a good idea to remove it?

Reply:
I guess it could be deleted

Reply:
Thus, change is fine. NTC.